docs/decisions/0008-pull-requests-code-review-guide.md

```diff
@@ -0,0 +1,103 @@
+# Pull Requests and Code Review Guide
+
+This is a guide not law.
+
+Use common sense.
+
+Have fun.
+
+## Protocol
+
+ 1. Open PR from your personal fork to a aquarist-labs repo.
+ 2. Assign to yourself. Set related Issue if it exists.
+ 3. Pick a reviewer. Ideally someone with domain knowledge. Pick more than one if the PR is particularly complex.
+ 4. Wait for review(s). Give about 2 days. If you haven't heard back after that or the PR is urgent ping reviewer(s).
+ 5. Discuss, address feedback until ≥ 1 reviewer(s) approve.
+ 6. Merge as soon as your work is approved
```

Comment:
Okay, I'd add a caveat here: if any one of the reviewers feels strongly about something, it would be good practice to check with them before merging the PR. Having one reviewer giving the thumbs up and another feeling strongly about a thumbs down, and merging nonetheless, is a bit of a dick move ;)

Reply:
> Also, as an example, take this PR: it's something that should be checked out by everyone. I've already signed-off on it. Should we merge it immediately, or wait for more reviews? :)

Good point. I should have pointed out that this is one of those consensus PRs

Reply:
> Okay, I'd add a caveat here: if any one of the reviewers feels strongly about something, it would be good practice to check with them before merging the PR. Having one reviewer giving the thumbs up and another feeling strongly about a thumbs down, and merging nonetheless, is a bit of a dick move ;)

You are right. Good points. I started the doc with "This is a guide not law. Use common sense. Have fun." for that reason. A  plea against dick moves :). 

How about a small addition for disapproval:

 5. Discuss, address feedback until ≥ 1 reviewer(s) approve. No reviewer(s) disapprove.